dsgrid/config/dataset_config.py

```diff
@@ -677,7 +678,9 @@ def get_value_columns(self) -> list[str]:
             case TableFormatType.PIVOTED:
                 return self.get_pivoted_dimension_columns()
             case TableFormatType.UNPIVOTED:
-                return [self.model.data_schema.table_format.value_column]
+                # TODO DT: is this acceptable
+                return [VALUE_COLUMN]
+                # return [self.model.data_schema.table_format.value_column]
```

Comment:
We were rarely using the user customization. I'm suggesting here that we should only ever use `"value"`. It will be easier to understand and code against. Users are already forced to encode column names with dsgrid dimension types, and so this doesn't seem like an inconvenience to me.

Reply:
I could see users wanting to name the column in whatever unit the value col represents, esp if a table is only going to have one type of value and one unit, but if we only want to support "value", it would be fine with me.